ConwaysGameOfLife.Tests/LocationShould.cs

```diff
@@ -57,5 +29,61 @@ public void WhenTwoLocationsInstantiated_NotBeEqual(int x1, int y1, int x2, int
 
             Assert.Equal(expected, result);
         }
+
+        [Fact]
+        public void WhenTwoListsOfLocationsSameOrderInstantiated_BeEqual()
+        {
+            var expected = new List<Location>(){
+                new Location(0, 0),
+                new Location(0, 1),
+                new Location(0, 2),
+                new Location(1, 0),
+                new Location(1, 2),
+                new Location(2, 0),
+                new Location(2, 1),
+                new Location(2, 2)
+            };
+
+            var result = new List<Location>(){
+                new Location(0, 0),
+                new Location(0, 1),
+                new Location(0, 2),
+                new Location(1, 0),
+                new Location(1, 2),
+                new Location(2, 0),
+                new Location(2, 1),
+                new Location(2, 2)
+            };
+
+            Assert.True(UniverseHelper.ListsOfLocationsAreEqual(expected, result));
+        }
+
+        [Fact]
+        public void WhenTwoListsOfCellsNotSameOrderInstantiated_BeEqual()

```

Comment:
Same goes re: `Instantiated`
```suggestion
        public void GivenTwoListsOfLocationsNotSameOrder_BeEqual()
```